crates/turborepo-lib/src/turbo_json.rs

```diff
@@ -0,0 +1,31 @@
+use std::{fs, path::Path};
+
+use anyhow::Result;
+use semver::{Version, VersionReq};
+use serde::Deserialize;
+
+#[derive(Deserialize, Debug)]
+#[serde(rename_all = "camelCase")]
+pub struct TurboJson {
+    pub turbo_version: String,
+}
+
+pub fn read_turbo_json(path: &Path) -> Result<TurboJson> {
+    let turbo_json_string = fs::read_to_string(path)?;
+    let turbo_json = serde_json::from_str(&turbo_json_string)?;
+
+    Ok(turbo_json)
+}
+
+impl TurboJson {
+    pub fn check_version(&self, version: &str) -> Result<bool> {
```

Comment:
I moved this into the `TurboJson` implementation since that seemed like a good shared home for it.